xml/html_title.xml

```diff
@@ -44,6 +44,7 @@
     <param pos="0" name="service.vendor" value="Amazon"/>
     <param pos="0" name="service.family" value="CloudFront"/>
     <param pos="0" name="service.product" value="CloudFront Load Balancer"/>
+    <param pos="0" name="service.cpe23" value="cpe:/a:amazon:amazon_cloudfront:-"/>
   </fingerprint>
```

Comment:
This as well as the new CPE for Akamai Ghost and a couple of AWS services is a result of requesting NIST to issue them. Thanks NIST!